docs/modules/operation/pages/events/mib.adoc

```diff
@@ -0,0 +1,40 @@
+
+= SNMP MIB Compiler
+
+Management Information Base (MIB) files are text files written in Abstact Syntax Notation.
+They describe Simple Network Management Protocol (SNMP) elements for specific devices.
+By default, {page-component-title} includes an extensive list of MIB definitions from a number of vendors.
+You will only need to import MIB files for unformatted events as part of configuring SNMP traps and collection.
+
+You can parse SNMP MIB definitions and create {page-component-title} event definitions from any `TRAP-TYPE` and/or `NOTIFICATION-TYPE` macros they contain through the UI.
+You can also generate data collection definitions from `OBJECT-TYPE` symbols defined in a MIB file.
+
+Two command-line utilities, `mib2events` and `mib2opennms`, are also available to help make event definitions from MIB definitions.
+The UI editor does at least as good a job as `mib2events`.
+We do not recommend using using `mib2opennms`, as it produces more bus errors than event definitions.
+
+[[mib-import]]
+
+To import a MIB file and create event definitions, follow these steps:
```

Comment:
```suggestion
To import an MIB file and create event definitions, follow these steps:
```

Reply:
I think people pronounce it "mib", not M.I.B., so it would be "a".